packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -141,6 +140,7 @@ const DomainPicker: FunctionComponent< Props > = ( {
 	areDependenciesLoading = false,
 	orderSubDomainsLast = false,
 	onUseYourDomainClick,
+	vendor,
```

Comment:
```suggestion
	vendor: domainSuggestionVendor = getDomainSuggestionsVendor(),
```

Reply:
Gonna go one step further and not use `domainSuggestionVendor` variable at all. https://github.com/Automattic/wp-calypso/pull/50761/commits/eebf36b5582e37d169569415cff38387eeca3410